docs/rewards/managing-rewards.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 2
+---
+
+# Managing Rewards 
+
+This guide will go through how to rewards are managed on the Archway Network by explaining the important fields and operations needed to track and receive rewards. 
+
+## Contract Metadata 
+Contract Metadata is the metadata that instructs the Archway Network who ones the contract and where rewards should be sent. The metadata has 3 specific parameters: 
+
+`contract_address` - The CosmWasm address of the contract 
+
+`owner_address` - The address of the contract's owner. This field can be both an account or contract address.
+
+`rewards_address` - The account address that will receive the contract's rewards. Tokens are sent to this address after a [withdrawal function](#withdrawing-awards) is executed. If not set, the contract will not receive rewards. 
+
+**All above addresses are bech32-encoded.**
+
+### Setting The Contract Metadata 
+
+The contract metadata must be set up by using the `MsgSetContractMetadata` transaction which must be signed by a contract admin. 
+
+```go  
+message ContractMetadata { 
+  string contract_address =  "archway14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9sy85n2u";
+  string owner_address = "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2";
+  string rewards_address = "archway12reqvcenxgv5s7z96pkytzajtl4lf2epyfman2";
+}
+``` 
```

Comment:
For clarity, it's better to show this information in a JSON object instead.

Reply:
The comment was not only about changing the code block language from `go` to `json` but also about the content. This is not a valid JSON :)